source/includes/Searching/Full/_intro.md

```diff
@@ -18,9 +18,19 @@ curl
 ```
 > Go [here](#full-searches_request-table) to se a full example of a request body and [here](#full-searches_response) to se a response body.
 
+<h5>EU CLUSTER:</h5>
 
 `POST https://api.cludo.com/api/v3/{CustomerId}/{EngineId}/search`
 
+<h5>US CLUSTER:</h5>
+
+`POST https://api-us1.cludo.com/api/v3/{CustomerId}/{EngineId}/search`
+
+<aside class="warning">
+If you are located on the US cluster <b>ALL</b> your HTTP requests should start with `https://api-us1.cludo.com/api/v3/`. Feel free to <a href="https://www.cludo.com/contact/">contact us</a> if you have any questions!
```

Comment:
We should only be using raw HTML when it's necessary, `**ALL**`. Also as it's already being emphasized, it's unnecessary to write it in CAPS as well. The same goes for the link `[contact us](https://www.cludo.com/contact/)` - but with that said we should probably add `target="_blank"` to it (which can only be achieved with raw HTML) to not take people away from the docs.